pythonforandroid/recipes/android/src/android/permissions.py

```diff
@@ -431,7 +434,7 @@ class _onRequestPermissionsCallback(PythonJavaClass):
     """Callback class for registering a Python callback from
     onRequestPermissionsResult in PythonActivity.
     """
-    __javainterfaces__ = ['org.kivy.android.PythonActivity$PermissionsCallback']
+    __javainterfaces__ = [ACTIVITY_NAMESPACE + '/' + ACTIVITY_CLASS_NAME + '$PermissionsCallback']
```

Comment:
I guess we got that wrong since the beginning , but it seems that both doted and slashed notation are working then, right?

Reply:
yes. exactly... i was in doubt, but checked with py-jnius docs and it says it must be '/'